packages/components/announcer/src/announcement.ts

```diff
@@ -0,0 +1,24 @@
+import { CSSResultGroup, LitElement, html } from 'lit';
+import styles from './announcer.scss.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'sl-announcement': Announcement;
+  }
+}
+
+export class Announcement extends LitElement {
+  /** @internal */
+  static override styles: CSSResultGroup = styles;
+
+  override render() {
+    return html`<slot></slot> `;
+  }
+
+  override connectedCallback(): void {
+    super.connectedCallback();
+    setTimeout(() => {
+      this.remove();
```

Comment:
Do this in `<sl-announcer>`, since that owns this element (see comment in announcer.ts)